app/src/main/java/pl/gdg/myapplication/MainActivity.kt

```diff
@@ -1,9 +1,9 @@
 package pl.gdg.myapplication
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/GDG-Wro/android_workshop_16/blob/9fced03aea4b1e029ea1462a9eb11d314df839dc/app/src/main/java/pl/gdg/myapplication/MainActivity.kt#L1"></span>The file /bitrise/src/app/src/main/java/pl/gdg/myapplication/MainActivity.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>